insights/parsers/lssap.py

```diff
@@ -0,0 +1,118 @@
+
+"""
+lssap - Command
+============
```

Comment:
@SteveHNH please include a `lssap.rst` file to include this module in the documentation build.

Reply:
Also, the preferred format for the heading is e.g. 'lssap - Command ``/usr/sap/hostctrl/exe/lssap`` with a matching length of double underline :-)